test/app.e2e-spec.ts

```diff
@@ -7,6 +7,7 @@ import { UserTests } from './userTests';
 import { PhoneTests } from './phoneTests';
 import { DepositTest } from './depositTests';
 import { LessonTests } from './lessonTests';
+import { PaymentTest } from './paymentTests';
```

Comment:
hmm, ja bym to nazwała `PaymentTests` żeby było jednakowo, co myślisz? deposit tu się trochę wyłamuje, to jakbyś miał ochotę to też możesz na mnogą przerobić :D